arkane/statmech.py

```diff
@@ -740,7 +740,7 @@ def create_hindered_rotor_figure(self, angle, v_list, cosine_rotor, fourier_roto
             ['$0$', '$\pi/4$', '$\pi/2$', '$3\pi/4$', '$\pi$', '$5\pi/4$', '$3\pi/2$', '$7\pi/4$', '$2\pi$'])
 
         self.hindered_rotor_plots.append((fig, rotor_index))
-        fig.close()
+        plt.close(fig)
```

Comment:
We use `fig.close()` in the kinetic_library_to_training_tools.py script as well (L208, L283). Do you think these should be replaced as well? If so, let add them to this PR

Reply:
Thanks for checking for other uses! That case is actually not related, since fig is a BytesIO object rather than a matplotlib figure.